.circleci/config.yml

```diff
@@ -626,15 +626,17 @@ workflows:
                 - master
     jobs:
       - build
-      - integration_tests:
-          requires: [build]
+      #- integration_tests:
```

Comment:
Can just remove the two lines I think

Reply:
I just wanted to see it pass on the PR branch and put it back into nightly ;)